omise/controllers/front/payment.php

```diff
@@ -0,0 +1,28 @@
+<?php
+if (! defined('_PS_VERSION_')) {
+    exit();
+}
+
+if (defined('_PS_MODULE_DIR_')) {
+    require_once _PS_MODULE_DIR_ . '/omise/classes/charge.php';
+}
+
```

Comment:
The reasons behind this design are, 
- To make code reusable on `OmisePaymentModuleFrontController`
- To make `Charge` class conform with `single responsibility` principle by, 1 class will take care only 1 thing (which is, `postProcess` function.)
- Prevent duplicating code in the future if we have more controller.
- And because I found that PrestaShop provided `postProcess()` & `redirect()` methods, that made our code more flexible to design.